frontend/src/pages/phone.page.tsx

```diff
@@ -43,6 +45,38 @@ const Phone: NextPage = () => {
     // ...there was a list of Relay numbers, but it was empty:
     relayNumberData.data?.length === 0;
 
+  const resendWelcomeSMSDismissal = useLocalDismissal(
+    `resend-sms-banner-${profile?.id}` ?? "unused"
+  );
+
+  const resendWelcomeText = !resendWelcomeSMSDismissal.isDismissed && (
+    <div className={styles["banner-wrapper"]}>
+      <Banner
+        title={l10n.getString("phone-banner-resend-welcome-sms-title")}
+        type="info"
+        cta={{
+          content: l10n.getString("phone-banner-resend-welcome-sms-cta"),
+          onClick: () => {
+            realPhoneData.resendWelcomeSMS();
+            toast(l10n.getString("phone-banner-resend-welcome-sms-toast-msg"), {
+              type: "success",
+            }),
+              resendWelcomeSMSDismissal.dismiss();
```

Comment:
I think this might technically work, but it's a bit confusing and I'm assuming it is a typo :)

```suggestion
            });
            resendWelcomeSMSDismissal.dismiss();
```